components/splitter/splitter-bar.component.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/LICENSE
+ */
+
+import { coerceCssPixelValue } from '@angular/cdk/coercion';
+import { ChangeDetectionStrategy, Component, computed, input, output, ViewEncapsulation } from '@angular/core';
+
+import { NzIconModule } from 'ng-zorro-antd/icon';
+import { getEventWithPoint } from 'ng-zorro-antd/resizable';
+import { NzSplitterCollapseOption } from 'ng-zorro-antd/splitter/typings';
+
+@Component({
+  selector: '[nz-splitter-bar]',
+  imports: [NzIconModule],
+  template: `
+    @if (lazy()) {
+      @let preview = active() && !!this.constrainedOffset();
+      <div
+        class="ant-splitter-bar-preview"
+        [class.ant-splitter-bar-preview-active]="preview"
+        [style.transform]="preview ? previewTransform() : null"
+      ></div>
+    }
+
+    <div
+      class="ant-splitter-bar-dragger"
+      [class.ant-splitter-bar-dragger-disabled]="!resizable()"
+      [class.ant-splitter-bar-dragger-active]="active()"
+      (mousedown)="resizeStartEvent($event)"
+      (touchstart)="resizeStartEvent($event)"
+    ></div>
+
+    @if (collapsible()?.start) {
+      <div class="ant-splitter-bar-collapse-bar ant-splitter-bar-collapse-bar-start" (click)="collapse.emit('start')">
```

Comment:
You already have a component function `resizeStartEvent` that emits `offsetStart` events.  
I would extract `collapse.emit(...)` into a component function too, to streamline where to look when browsing code.